src/runtime/type.ts

```diff
@@ -832,3 +834,17 @@ export class HandleType extends Type {
     return {tag: this.tag};
   }
 }
+
+export class DummyType extends Type {
```

Comment:
Suggest adding a brief comment on the purpose of this -- why it exists, when it might be used/useful.

Reply:
I am a bit baffled but this. Can we maybe reuse the type of the original proxy when creating a dummy? Having a type in the type system that is there just to satisfy hot code reloading looks like something we should not do if we can avoid.

Reply:
Sounds good, deleted this type and just use whatever type of the original StorageProxy.